crates/hc_sandbox/tests/cli.rs

```diff
@@ -567,13 +585,20 @@ fn get_sandbox_command() -> Command {
     Command::new(get_target("hc-sandbox"))
 }
 
-async fn get_launch_info(mut child: Child) -> LaunchInfo {
+async fn get_launch_info(child: &mut Child) -> LaunchInfo {
     let stdout = child.stdout.take().unwrap();
     let mut lines = BufReader::new(stdout).lines();
     while let Ok(Some(line)) = lines.next_line().await {
-        println!("@@@@@-{line}-@@@@@");
+        println!("@@@-{line}-@@@");
         if let Some(index) = line.find("#!0") {
             let launch_info_str = &line[index + 3..].trim();
+
+            tokio::task::spawn(async move {
```

Comment:
Dropping stdout before the sandbox is finished causes annoying broken pipe errors and if it logs something useful then we'll miss it. Just try to capture as much as the program outputs.

Reply:
For some reason, this won't work for me on Windows. I spent some time debugging but I really don't understand why code that we have in several places in production code is causing problems here.